wallet/src/Pos/Wallet/Web/State/Storage.hs

```diff
@@ -636,21 +637,40 @@ createWallet cWalId cWalMeta isReady curTime = do
     let info = WalletInfo cWalMeta curTime curTime NotSynced noSyncStatistics mempty isReady
     wsWalletInfos . at cWalId %= (<|> Just info)
 
--- | Add new address given 'CWAddressMeta' (which contains information about
--- target wallet and account too).
+-- -- | Add new address given 'WAddressMeta' (which contains information about
+-- -- target wallet and account too). If the input account is /not/ there, creates it.
```

Comment:
Double `--` ? Probably good to fix that.

Reply:
arf :disappointed: